### DISCUSSION TITLE
Other Character Placement

### DISCUSSION THREAD

**Discussion by bigo93** [RobertMacdonald7/BlishHud-PlayerMarker#16] [Ideas]
This would be very useful in WvW if you could place the marker on the commander's character, or another player's character instead of your own.
Any way of doing that?

**Comment by RobertMacdonald7** [comment]
I remember looking into this previously, and unfortunately I don't think the current APIs support tracking the position of other characters.

If you haven't tried it already, you could use "Set Personal Target" on the commander, that'll put a fairly obvious marker above them that you can track

**Comment by bigo93** [comment]
Shame.
Yeah already tried the personal target, but it still gets lost in with all the skills when it's a big zerg fight.
But your mod allows you to change the size as well.

And unfortunately the Commander Markers one still needs you to tag up as a commander to use.
